foundationdb/manifest.json

```diff
@@ -0,0 +1,170 @@
+{
+  "display_name": "FoundationDB",
+  "maintainer": "datadog-integrations@edument.cz",
+  "manifest_version": "1.0.0",
+  "name": "foundationdb",
+  "metric_prefix": "foundationdb.",
+    "metric_to_check": [
```

Comment:
metric to check should include just one or two metrics. This field is used to display, on the integration tile if the integration is enabled and working properly so it should be a metric that is always emitted.

Reply:
Makes sense; changed.